cccat/events/event_handlers.py

```diff
@@ -25,7 +27,8 @@ def handle_ping(data: dto.JSON) -> None:
 def handle_cat_created(data: dto.JSON) -> None:
```

Comment:
[question] Can you explain why CI is failing and what we can do to fix that?

Reply:
@Chandan-Baba CI is failing because the coverage is not sufficient. The result of the CI shows which lines are not covered by test to help what cases should be added.
https://github.com/Chandan-Baba/tech-stack-training/pull/2/checks?check_run_id=3623821663
![image](https://user-images.githubusercontent.com/45156412/134133106-d86ea976-c75c-43f8-ac24-6b9fa6e677d3.png)